app/Services/PlanetService.php

```diff
@@ -1462,6 +1520,7 @@ public function getPlanetBasicIncome(): Resources
      */
     private function updateResourceProductionStatsInner(Resources $production_total, int|float $energy_production_total, int|float $energy_consumption_total, bool $save_planet = true): void
     {
+
```

Comment:
It looks like nothing changed in this method `updateResourceProductionStatsInner()` except adding and removing a newline, can you revert the change on line 1523 and line 1482?

Reply:
> It looks like nothing changed in this method `updateResourceProductionStatsInner()` except adding and removing a newline, can you revert the change on line 1523 and line 1482?

I have updated the code again for your reference. Since the production calculations are added to the base production, you can see it in PlanetService::getPlanetBasicIncome().